### DISCUSSION TITLE
Hiding specific panes or emulating stacked panes

### DISCUSSION THREAD

**Discussion by meepleek** [wezterm/wezterm#5766] [Q&A]
### What Operating System(s) are you running on?

Linux Wayland

### Which Wayland compositor or X11 Window manager(s) are you using?

kwin & hyprland

### WezTerm version

wezterm 20240623-141056-2d51ab22

### Ask your question!

I'd like to have a status pane that shows 1 active pane (eg. tasklist, bacon clippy, bacon test) out of multiple and I can switch between the shown pane either by hiding the remaining panes  (like zoom, but for a given pane group/split, not the whole tab) or by stacking the panes. Is that achievable?  

I don't want just close & reopen the panes as those could be scrolled and that might cause needles work.   
I think I might be able to use `AdjustPaneSize` to stack the panes, but for that to be useful a pane title would be preferable to just showing 1 line from the pane. Alternatively `pane:move_to_new_window` seemed kinda useful, but I don't see a way to move it to a specific window or swap panes (so have a hidden/stash tab with the inactive panes).

**Comment by 00sapo** [comment]
See my solution: https://github.com/wezterm/wezterm/discussions/3779#discussioncomment-14879236
